frontend/interfaces/integration.ts

```diff
@@ -0,0 +1,29 @@
+export interface IJiraIntegration {
+  url: string;
+  username: string;
+  password: string;
+  project_key: string;
+  enable_software_vulnerabilities?: boolean;
+  integrationIndex?: number;
```

Comment:
Integrations are in an array that's not indexed, mapped to make the index the unique key so we can edit and delete specific indices from the FE